index.rst

```diff
@@ -71,80 +40,85 @@ A general overview of the development flow, as viewed from the user, is as follo
 
-    - Create a job, which is runnable by the Queue, to perform these tasks.
+    - Create a script, which is runnable by the scriptQueue, to perform these tasks.
 
       Note that at this point the utilities may still be rough, and certain functionality might be better accomplished
       in lower classes (e.g. attcs) but that functionality does not yet exist.
 
     - Request new functionality in lower-level control classes (e.g. attcs)
 
-    - Migrate/evolve the utilities and Job to a production level for regular use with the Queue
+    - Migrate/evolve the utilities and script to a production level for regular use with the scriptQueue
 
 How the workflow moves through the various repositories is represented by the following diagram, where each of the
 sections is discussed in detail below.
 
-.. figure:: _static/notebook_and_queueJob_workflow_diagram.jpg
+.. figure:: _static/Notebook_and_script_workflow_v2.jpg
     :width: 600px
     :align: center
     :alt: Workflow Summary
 
     A visual overview of the workflow process, starting from an original idea first demonstrated in a Jupyter notebook.
 
-The two-tiered development strategy is being adopted to facilitate maximum flexibility to implement changes on
+The three-tiered development strategy is being adopted to facilitate maximum flexibility to implement changes on
 short timescales. The staging and development areas gives users full range to simultaneously operate across multiple
 versions/branches with unhindered flexibility. This is especially important for commissioning activities where rapid
 code changes are required and temporary implementation is required which will break other people's code in the same
 area.
 
-The production area is to provide a single code base where all code in that area (on the master/develop branches) is
+The production area is to provide a single code base where all code in that area (on the master and develop branches) is
 fully functional at all times. This will be most important during operations when the codebase will be less dynamic
 but will also be useful for different commissioning teams that will circle through and want an established and stable
 repository to start from without having to diagnose what changes the previous group has made to facilitate their
 goals. This area of the codebase will also require larger amounts of
 testing in order to ensure that changes made in one area do not result in breaking code in another.
 
 
-
-.. note::
-
-    This technote avoids the use of the term "scripts" and also assumes the current scriptQueue has been renamed to
-    "Queue". The renaming of the scripts and scriptQueue is being discussed elsewhere. The document will be updated
-    upon resolution of the ticket.
-
-
 .. _notebooks:
 
 Jupyter notebooks
 =================
 Jupyter notebooks (henceforth referred to as notebooks) will be the primary tool used in system verification
```

Comment:
I think you referred to jupyter notebooks as notebook before.

Reply:
Ya, mostly the first paragraph. Corrected